pkg/agent/dozer/bcm/plan.go

```diff
@@ -465,6 +470,99 @@ func planFabricConnections(agent *agentapi.Agent, spec *dozer.Spec) error {
 	return nil
 }
 
+func planMeshConnections(agent *agentapi.Agent, spec *dozer.Spec) error {
+	// TODO: Decide whether we want:
```

Comment:
I think we should keep fabric-mode as spine-leaf for simplicity as overall logical approach is the same and just validate that only one of (fabric,mesh) connection type is present. We can discuss more if you think it involves more pain

Reply:
let's drop that comment for now